internal/app/admin/http/cache_test.go

```diff
@@ -699,6 +505,42 @@ func TestAdminServer_CacheDumpHandler_WildcardSuffix_NotFound(t *testing.T) {
 	}
 }
 
+func verifyCacheOutput(t *testing.T, rr *httptest.ResponseRecorder, cdsFile string, ldsFile string) {
+	var actualResponse map[string]interface{}
+	err := json.Unmarshal(rr.Body.Bytes(), &actualResponse)
+	assert.NoError(t, err)
+
+	filecontentsCds, err := ioutil.ReadFile(cdsFile)
+	assert.NoError(t, err)
+	var expectedCdsResponse map[string]interface{}
+	err = json.Unmarshal(filecontentsCds, &expectedCdsResponse)
+	assert.NoError(t, err)
+
+	filecontentsLds, err := ioutil.ReadFile(ldsFile)
+	assert.NoError(t, err)
+	var expectedLdsResponse map[string]interface{}
+	err = json.Unmarshal(filecontentsLds, &expectedLdsResponse)
+	assert.NoError(t, err)
+
+	actualCacheResponse := actualResponse["Cache"].([]interface{})
+	assert.Equal(t, len(actualCacheResponse), 2)
+	var actualLdsResponse map[string]interface{}
+	var actualCdsResponse map[string]interface{}
+
+	for _, c := range actualCacheResponse {
+		if c.(map[string]interface{})["Key"] == "test_lds" {
+			actualLdsResponse = c.(map[string]interface{})
+		} else {
+			actualCdsResponse = c.(map[string]interface{})
+		}
+	}
+
+	assert.Equal(t, expectedLdsResponse["Key"], actualLdsResponse["Key"])
+	assert.Equal(t, expectedCdsResponse["Key"], actualCdsResponse["Key"])
+	assert.Equal(t, expectedLdsResponse["Resp"], actualLdsResponse["Resp"])
+	assert.Equal(t, expectedCdsResponse["Resp"], actualCdsResponse["Resp"])
```

Comment:
one last thing...can we add a check that `Requests`  array has len 1

Reply:
same with ExpirationTime

Reply:
Did it for Requests. Can't with ExpirationTime because it is not an array/slice. I added a NotNil check though